### PR TITLE
Patch to handle Array Declaration #237

### PR DESCRIPTION
**Pull request by frodrigo**
Patch to handle Array Declaration #237

### CODE REVIEW

File: lib/grape-swagger/doc_methods.rb [hunk 1]
```diff
@@ -47,6 +47,12 @@ def parse_params(params, path, method)
                           'double'
                         when 'Symbol'
                           'string'
+                        when /^\[(.*)\]$/
```
Comment (dblock) [inline lib/grape-swagger/doc_methods.rb]: Maybe use a named field here, like type. That avoids having a `last_match[1]` and is probably clearer.

**Comment by frodrigo** [comment]
If it's OK I can squash them.

File: lib/grape-swagger/doc_methods.rb [hunk 2]
```diff
@@ -47,6 +47,12 @@ def parse_params(params, path, method)
                           'double'
                         when 'Symbol'
                           'string'
+                        when /^\[(?<type>.*)\]$/
+                          items[:type] = Regexp.last_match[:type].downcase
+                          if PRIMITIVE_MAPPINGS.key?(items[:type])
+                            items[:type], items[:format] = PRIMITIVE_MAPPINGS[items[:type]]
```
Comment (dblock) [inline lib/grape-swagger/doc_methods.rb]: I re-read this and find it a bit strange (or at least looks like a side-effect) that we're assigning `items[]` inside the case statement. Feels like this logic belongs below where we deal with `items`? I am not sure though, what do you think?

Comment (frodrigo) [inline lib/grape-swagger/doc_methods.rb]: Later we deal with parsed_params[:type], not items[:type], items just affected into parsed_params[:items].

**Comment by dblock** [comment]
Please squash and update CHANGELOG. Thanks.

**Comment by frodrigo** [comment]
Squashed + changelog done

**Comment by dblock** [comment]
Merged.

**Pull request merged**